executor/linux/build.go

```diff
@@ -164,7 +165,14 @@ func (c *client) PlanBuild(ctx context.Context) error {
 			continue
 		}
 
-		c.Logger.Infof("pulling %s %s secret %s", secret.Engine, secret.Type, secret.Name)
+		// only pull in secrets that are set to be pulled in at the start
+		if strings.EqualFold(secret.Pull, constants.SecretPullStep) {
+			c.Logger.Infof("SecretPullBuild")
```

Comment:
should we remove this debug line, or adjust it to make it look consistent with our other logs?

Reply:
I adjusted it!

Reply:
what was the adjustment? i don't think this line was adjusted.

Reply:
it should be there now, something weird was going on but I fixed it